doc/doxygen/install/install-win.doxygen

```diff
@@ -269,39 +269,73 @@
+    \endcode
   
   Example:
   \code
   MSBuild.exe OpenMS_host.sln
   \endcode
 
-  @note Since the call can be a bit lengthy, there is a batch file in
-  <tt>OpenMS/tools/build.bat</tt> and in the root of your build tree (copied
-  there by cmake for convenience), which allows to build the most important
-  targets (TOPP, UTILS, Gui, Tests, Doc) in Release or Debug using very short
-  notation. Call it without arguments to see its usage help text. For example
-  to build only the %OpenMS library in release mode, call
-
-  \code
-  build OpenMS r
-  \endcode
+    @note Since the call can be a bit lengthy, there is a batch file in
+    <tt>OpenMS/tools/build.bat</tt> and in the root of your build tree (copied
+    there by cmake for convenience), which allows to build the most important
+    targets (TOPP, UTILS, Gui, Tests, Doc) in Release or Debug using very short
+    notation. Call it without arguments to see its usage help text. For example
+    to build only the %OpenMS library in release mode, call
 
+    \code
+    build OpenMS r
+    \endcode
 
+  @section install_win_create_installer Creating an installer 
+  
+    <ol>
+      <li> Download <a href="https://sourceforge.net/projects/nsis/" target="_blank">NSIS</a> (we tested nsis-3.06.1 successfully) and extract it somewhere
+
+      <li> Install the <a href="https://nsis.sourceforge.io/Special_Builds" target="_blank">large-string-build NSIS installer</a> on top (just extract into the same directory as NSIS; and use a matching version!).
+           This is required for PATH manipulation of up to 8k length; the default NSIS handles up to 1k strings, which might be too short and potentially leads to a broken PATH environment variable.
+      
+      <li> Install the <a href="https://github.com/SuperPat45/UltraModernUI" target="_blank">UltraModernUI-plugin</a> on top (just extract into the same directory as NSIS). 
+           We tested <a href="https://github.com/SuperPat45/UltraModernUI/releases/tag/2.0b6" target="_blank">version 2.0b6</a> successfully.
+
+      <li> Add the NSIS folder (which contains 'makensis.exe') to your PATH, so CMake can find it.
+
+      <li> Checkout the <a href="https://github.com/OpenMS/THIRDPARTY" target="_blank">THIRDPARTY GitHub Repository</a> and flatten the structure for the target platform, e.g. copy all subdirectories in <tt>./all/</tt> to <tt>./Windows/64bit/</tt>.
+
+      <li> Configure %OpenMS as usual with additional flags <tt>SEARCH_ENGINES_DIRECTORY</tt> (for the flattened THIRDPARTY directory) and <tt>PACKAGE_TYPE="nsis"</tt>, 
+      e.g.
+      \code
+      cd <build-tree>
+      cmake ... -DSEARCH_ENGINES_DIRECTORY=C:\dev\THIRDPARTY\Windows\64bit -DPACKAGE_TYPE="nsis" <path-to-source-tree>
+      \endcode
+      
+      <li> Build all targets (incl. 'doc' and 'doc_tutorials') in Release mode (copy the Qt5 plugin to the <code>[OpenMS_build]\bin\Release</code> and <code>[OpenMS_build]/doc/doxygen/parameters/Release/</code> directories first)
```

Comment:
Would be cool if we could get rid of this at some point. Should be relatively easy to do with CMake right?
I.e. as a pre-build step for doc or doc_tutorials to copy the files. Or just call findQt there again. Or pass the location down to this included script.

Reply:
Ah no, sorry, the problem is the tools to find the dependency at runtime before being packaged.
Hm for this only the auto-copy option or adding the Qt folder to the path via CMake are the remaining options.